src/pages/ipfs/IpfsList.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { ReactNode } from 'react';
 import { Link } from 'react-router-dom';
+// TODO: Remove semantic here later on when this is tackled and use directly tailwind
```

Comment:
Goal would be to remove semantic ui and use as much as possible tailwind

Reply:
I would remove semantic-ui totally and use only tailwind

Reply:
Yeah saw that they do have tailwind-ui stuff already premade, but wondering if we can stick to just their basic stuff in tailwindcss
I think that's probably what @Netherdrake was referring too the other day about pricing

Reply:
Well if you ask me I would definitely go with premade stuff :) Just look at all those awesome components/pages, amazing